src/SendGrid/SendGridClient.cs

```diff
@@ -46,6 +49,11 @@ public class SendGridClient : ISendGridClient
         private HttpClient client;
 
         /// <summary>
+        /// Acceptable format for date in the API
```

Comment:
This is one of the "acceptable format for date", but not the only one. There are at least three other formats that I'm aware are used by different endpoints in the SendGrid API (see [here](https://github.com/sendgrid/sendgrid-csharp/issues/193) for more details).